tox.ini

```diff
@@ -9,6 +9,7 @@ deps =
     nose
     nose-exclude
     unittest2
+    -egit+https://github.com/google/oauth2client.git#egg=oauth2client
```

Comment:
We're going to need to pin `oauth2client>=2.1.0` (or whatever version gets released with this feature) somewhere, no?

Otherwise, WNTL?